consensus/propagation/reactor.go

```diff
@@ -49,6 +49,9 @@ type Reactor struct {
 }
 
 func NewReactor(self p2p.ID, tracer trace.Tracer, store *store.BlockStore, options ...ReactorOption) *Reactor {
+	if tracer == nil {
+		tracer = trace.NoOpTracer()
+	}
```

Comment:
no changes needed

but typically I think we should avoid passing nil for anything. instead we should pass the nooptracer